File: 217. Contains Duplicate/217. Contains Duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let uniqueCharacters = new Set();
    for (let i = 0; i < nums.length; i++)
      if (uniqueCharacters.add(nums[i]).size === i)
        return true
    return false
};

File: 217. Contains Duplicate/217. Contains Duplicate.java
import java.util.HashSet;
import java.util.Set;

class Main {
  public static void main(String[] args) {
    int[] nums = {1,2,3,1};
    System.out.println(containsDuplicate(nums));
  }
  public static boolean containsDuplicate(int[] nums) {
    Set<Integer> uniqueIntegers = new HashSet<>();
    for (int i : nums)
     if (!uniqueIntegers.add(i))
      return true;
    return false;
  }
}